extensions/cornerstone/src/contextProviders/ViewportActionCornersProvider.tsx

```diff
@@ -35,8 +35,8 @@ export const ViewportActionCornersContext = createContext(DEFAULT_STATE);
 export function ViewportActionCornersProvider({ children, service }) {
   const viewportActionCornersReducer = (state, action) => {
     switch (action.type) {
-      case 'SET_ACTION_COMPONENT': {
-        const { viewportId, id, component, location, indexPriority = 0 } = action.payload;
+      case 'ADD_ACTION_COMPONENT': {
+        const { viewportId, id, component, location, indexPriority } = action.payload;
```

Comment:
The location should come from the component and NOT the add command.  The reason is that one can then update the component to work in a different location if desired, and it will just flow through properly.  For example, a window level component which is down at the bottom of the screen is perfectly fine, rather than one at the right.

Reply:
I'm not sure about this, let's talk

Reply:
I decided to do this in next version not this one https://github.com/OHIF/Viewers/issues/4457